bedrock/base/templates/includes/site-footer.html

```diff
@@ -40,6 +40,7 @@
                 <ul>
                   <li><a href="https://twitter.com/mozilla" data-link-type="footer" data-link-name="Twitter (@mozilla)">{{ _('Twitter') }}<span> (@mozilla)</span></a></li>
                   <li><a href="https://www.facebook.com/mozilla" data-link-type="footer" data-link-name="Facebook (Mozilla)">{{ _('Facebook') }}<span> (Mozilla)</span></a></li>
+                  <li><a href="https://www.instagram.com/mozillagram/" data-link-type="footer" data-link-name="Instagram (@mozillagram)">{{ _('Instagram') }}<span> (@mozillagram)</span></a></li>
```

Comment:
/cc @peiying2 @flodolo - this adds a new string, but I'm guessing "Instagram" is unlikely to be translated by most locales.